diagnostics/minspline.py

```diff
@@ -10,7 +10,7 @@
 import numpy as np
 from openest.models.curve import CubicSplineCurve
 
-def quadratic(aa, bb, cc):
+def quadratic_formula(aa, bb, cc):
```

Comment:
isn't this an old form of the quadratic formula? we should only used eisntein et al. 's version!